.eslintrc.js

```diff
@@ -116,6 +116,7 @@ module.exports = {
     'no-restricted-globals': 'warn',
     'react/no-array-index-key': 'warn',
     'react/jsx-curly-brace-presence': 'warn',
+    'react/function-component-definition': 'off',
```

Comment:
I think it is ok to create a Functional Component with an arrow function. What do you think, guys?